launcher/cli/main.go

```diff
@@ -46,6 +46,9 @@ func Main() {
 	RootCmd.PersistentFlags().Bool("log-to-file", true, "Also write logs to {data-dir}/dfuse.log.json ")
 	RootCmd.PersistentFlags().CountP("verbose", "v", "Enables verbose output (-vvvv for max verbosity)")
 
+	RootCmd.PersistentFlags().String("log-level-switcher-addr", "localhost:1065", "If non-empty, the process will listen on this address for json-formatted requests to change different logger levels (see DEBUG.md for more info)")
```

Comment:
Missing the `listen` suffix somewhere, it's kind of our convention now from what I recall.